src/gui/dialogs/DialogStateful.hpp

```diff
@@ -83,12 +78,12 @@ class IDialogStateful : public IDialog {
     bool Change(uint8_t phase, uint8_t progress_tot, uint8_t progress); // = 0; todo should be pure virtual
     virtual ~IDialogStateful();
 
-    static constexpr rect_ui16_t get_radio_button_size() {
-        rect_ui16_t rc_btn = { 0, 32, 240, 320 - 96 }; //msg box size
-        rc_btn.y += (rc_btn.h - 40);                   // 30pixels for button (+ 10 space for grey frame)
-        rc_btn.h = 30;
-        rc_btn.x += 6;
-        rc_btn.w -= 12;
+    static rect_ui16_t get_radio_button_size() {                                // cannot be const(expr)
```

Comment:
It should be constexpr
problem is gui_defaults is written in C and will be rewritten to C++

Reply:
I recommend modifying gui_defaults to have `#ifdef __cplusplus` version

Reply:
I would leave it for different PR focused on C -> C++ transition.

Reply:
BFW-954 was made.

Reply:
However, it's duplicit work to review this before 889 is merged.